payment_redsys/models/payment_acquirer.py

```diff
@@ -105,7 +105,7 @@ def _get_website_url(self):
             or self.env.context.get("website_id")
             and self.env["website"].browse(self.env.context["website_id"])
         )
-        domain = website and website.domain or ""
+        domain = website and website.domain
```

Comment:
Por qué quitar el `or ""`. Así, puedes tener un `False` pululando por ahí que dé error.

Reply:
Eso diría que se tendría que haber preguntado en el PR original de https://github.com/OCA/l10n-spain/pull/3777/files 😝  ya que realizaron ellos el cambio. Podemos volverlo a añadir en esta versión si es necesario.

Reply:
Ya, ahí no revisé yo como vi 3 aprobados y lo único que hice fue fusionar, pero no debería quitarse. Si no, no tiene sentido tampoco el `and` y bastaría con `website.domain`. Restituidlo por favor.

Reply:
te parece que se haga en un segundo commit por si lo quieres pasar a las superiores?

Reply:
No merece la pena un commit aparte.